modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -24,11 +24,23 @@ interface RenderedDialog<T extends Dialog.DialogData> {
   readonly instanceApi: Dialog.DialogInstanceApi<T>;
 }
 
+const getInlineDialogSizeClasses = (size: Dialog.DialogSize): string[] => {
+  switch (size) {
+    case 'large':
+      return [ 'tox-dialog--width-lg--inline' ];
```

Comment:
This hasn't been updated to match what is in `dialog.less`. Could we have a test to check the dialog height to protect against this?

Reply:
Fixed

Checking any conflict with https://ephocks.atlassian.net/browse/TINY-9941...

Reply:
added test